packages-common/pluggable-widgets-generator/generators/app/templates/packages/package.js.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "<%- packageName %>",
+  "widgetName": "<%- widgetName %>",
+  "version": "<%- version %>",
+  "description": "<%- description %>",
+  "copyright": "<%- copyright %>",
+  "author": "<%- author %>",
+  "config": {
+    "widgetPath": "<%- projectPath %>/widgets",
+    "projectPath": "<%- projectPath %>/",
+    "mendixHost": "http://localhost:8080",
+    "developmentPort": "3000"
+  },
+  "packagePath": "<%- packagePath %>",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:server --open",
+    "dev": "pluggable-widgets-tools start:web",
+    "build": "pluggable-widgets-tools build:web",
+    "lint": "pluggable-widgets-tools lint",
+    "lint:fix": "pluggable-widgets-tools lint:fix",<% if (unitTests || e2eTests) { %>
+    "pretest": "npm run lint && npm run build",<% } %><% if (unitTests) { %>
+    "test": "pluggable-widgets-tools test:unit --no-cache --ci<% if (e2eTests) { %> && npm run test:e2e<% } %>",
+    "test:unit": "pluggable-widgets-tools test:unit --coverage",<% } %><% if (e2eTests) { %>
+    "test:e2e": "export URL=https://badge.mxapps.io || set URL=https://badge.mxapps.io && pluggable-widgets-tools test:e2e",
+    "test:e2e:dev": "export DEBUG=true || set DEBUG=true && pluggable-widgets-tools test:e2e",<% } %>
```

Comment:
Please use make use of `cross-env` preventing an other console error, and it looks nicer.
`'`json
"test:e2e": "cross-env URL=https://badge.mxapps.io pluggable-widgets-tools test:e2e:ts",
 "test:e2e:dev": "cross-env DEBUG=true pluggable-widgets-tools test:e2e:ts",
```
This probably requires to add a dependency to the widget tools

Reply:
Then, most probably, you can remove that dependency from the root of the repository

Reply:
Thanks for your input Andries! 😃 

The scripts have been converted to the following:

```
 "test:e2e": "pluggable-widgets-tools test:e2e --url \"https://badge.mxapps.io\""
 "test:e2e:dev": "pluggable-widgets-tools test:e2e --debug"
```

PIW-tools is still backwards compatible with older generators that use environment variables.